monitoring/prometheus/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "sli-metric-sample-node",
+  "version": "1.0.0",
+  "description": "",
+  "main": "app.js",
+  "scripts": {
+    "start": "node app.js",
+    "test": "mocha -- system-test/*.test.js --timeout=5000 --exit"
+  },
+  "author": "Yuri Grinshteyn",
+  "license": "Apache-2.0",
+  "dependencies": {
+    "express": "^4.17.1",
+    "lint": "^0.7.0",
+    "node-stopwatch": "^0.0.1",
+    "prom-client": "^12.0.0",
+    "request": "^2.88.2",
```

Comment:
Request is deprecated. Can you use `node-fetch` instead please.

Reply:
Request is there by mistake - removed it.